### ISSUE TITLE
Badge appears on app icons of different flavours

### ISSUE THREAD

**Issue by yu-t** [leolin310148/ShortcutBadger#228]
This issue happens for me for now only on LG g5 (7.0)
There're several apps installed with the same package name and the same name of launcher activity.
When I call "ShortcutBadger.applyCount(this, 7);" I see that the badge is applied to the app icons of all applications.

**Comment by rsanjuan87** [comment]
That's right, I have 2 flavours (Dev, Rel) instaled in my personal device and when I open a flavour it sets it's own counter to boths icons

**Comment by yu-t** [comment]
@rsanjuan87 I mean not Dev/Rel flavours, but 2 different flavours (apps) with the same name of the launcher activity. And it doesn't show counter on both of them on Samsung s6, for example. Only on LG g5. It seems to be a launcher issue.

**Comment by jkasten2** [comment]
@yu-t  @rsanjuan87 It's a know bug in the LG Launcher. All apps with the same Activity name (including the namespace) will share the same badge value.
